net-libs/tox/tox-0.2.11.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils systemd
```

Comment:
Switch to *cmake*.eclass instead, and remember to update function calls below. Looks good otherwise!